src/lib/tuya.ts

```diff
@@ -429,6 +430,91 @@ export const valueConverterBasic = {
     trueFalse: (valueTrue: number | Enum) => {
         return {from: (v: number) => v === valueTrue.valueOf()};
     },
+    color1000: () => {
```

Comment:
@kirovilya will this converter be used? I propose to not add it here until it is removed from `legacy.ts` (`silvercrest_smart_led_string`)

Reply:
OK. I can temporarily comment out this converter until the next improvements.
I'll leave only the brightness and color temperature in "light" extend, without color.